File: web3rpc/rpc-specs/code-samples/python/debug/blockchainInspection/getModifiedAccountsByNumber.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

startBlockNum = 171904
endBlockNum = 172160

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.get_modified_accounts_by_number(startBlockNum, endBlockNum)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/blockchainInspection/getModifiedAccountsByNumber.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const startBlockNum = 171904;
  const endBlockNum = 172160;

  provider.debug
    .getModifiedAccountsByNumber(
      startBlockNum,
      { endBlockNum },
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
